reference/library/src/morpheus-master/sass/modules/tool/_base.scss

```diff
@@ -29,23 +29,36 @@
 		}
 		a{
 			color: lighten( $text-color, 70% );
+			margin-right: 0.25em;
+
 		}
 	}
 
 	.#{$namespace}toolTitleNav__link{
-		vertical-align: bottom;
+		vertical-align: 0%;
+		color: lighten( $text-color, 70% );
+		text-decoration :none;
+		margin-right: 0.25em;
 		@include icon();
 		&--directurl{
-			@include base-sprite(link);
+			@extend .fa;
+			@extend .fa-link;
+			@extend .fa-lg;
 		}
 		&--help-popup{
-			@include base-sprite(help);
+			@extend .fa;
+			@extend .fa-question;
+			@extend .fa-lg;
 		}
 		&--edit{
-			@include base-sprite(edit);	
+			@extend .fa;
+			@extend .fa-pencil;
+			@extend .fa-lg;	
 		}
 		&--reset{
-			@include base-sprite(reload);	
+			@extend .fa;
+			@extend .fa-home;
```

Comment:
We can use that but, there was a debate on what to use here, Sakai for long time used the refresh button but the actual function of this button is more of take me to the portal/tool home than refresh. When Mark did usability study in NYU he has done this change to make it as home in his initial design, I personally agree with his idea too, but we can discuss this during  next Thursday feedback session to take more input on what it should be by default.